test/integration/generated/readmeFileChecker/src/keyVaultClient.ts

```diff
@@ -36,6 +36,7 @@ import {
   KeyVaultClientGetDeletedSecretsNextResponse
 } from "./models";
 
+/** @internal */
```

Comment:
What triggered this change?

Reply:
hide-client is set in the md file: https://github.com/Azure/autorest.typescript/blob/02eefa6ee665805141b9291e71a2f719d690fa4d/test/integration/swaggers/keyvaults-secrets.md so I am not sure why it was not triggered before.